### PR TITLE
MINOR: Fix NPE handling unknown APIs in NodeApiVersions.toString

### PR DESCRIPTION
**Pull request by hachikuji**

### CODE REVIEW

**Comment by hachikuji** [comment]
cc @cmccabe

**Comment by ijuma** [comment]
@hachikuji, can you please add a test?

File: clients/src/main/java/org/apache/kafka/clients/NodeApiVersions.java [hunk 1]
```diff
@@ -47,7 +46,7 @@
      * @return A new NodeApiVersions object.
      */
     public static NodeApiVersions create() {
-        return create(Collections.EMPTY_LIST);
+        return create(Collections.<ApiVersion>emptyList());
```
Comment (cmccabe) [inline clients/src/main/java/org/apache/kafka/clients/NodeApiVersions.java:49]: Oh, interesting.  I guess this is considered an unchecked conversion, so we should be using the emptyList() form.  Thanks for finding this.

**Comment by cmccabe** [comment]
LGTM (non-binding).  Thanks for adding the test.

It looks like this turned up in the findbugs warnings as well.

**Review by ijuma — APPROVED** [review]
Thanks for the PR, LGTM.

**Comment by ijuma** [comment]
Merged to trunk and 0.10.2 branches.

**Pull request closed without merging**